src/test/java/redis/clients/jedis/tests/JedisPoolTest.java

```diff
@@ -42,56 +40,64 @@ public void checkConnections() {
   @Test
   public void checkCloseableConnections() throws Exception {
     JedisPool pool = new JedisPool(new JedisPoolConfig(), hnp.getHost(), hnp.getPort(), 2000);
```

Comment:
@sazzad16 I think the JedisPool is also closable

Reply:
@gkorland we need for `assertTrue(pool.isClosed());` below